setup.py

```diff
@@ -57,7 +57,7 @@ def build_extension(self, ext):
             build_args += ["--", "/m"]
         else:
             cmake_args += ["-DCMAKE_BUILD_TYPE=" + cfg]
-            build_args += ["--", "-j{}".format(max(1, os.cpu_count() - 1))]
+            build_args += ["--", "-j{}".format(max(1, len(os.sched_getaffinity(0)) - 1))]
```

Comment:
Why do we override the defaults? There should be env vars controlling the nr of make/ninja processes, no?

Reply:
Not sure if there are dedicated variables controlling this.